app/javascript/react/screens/App/Overview/screens/MappingWizard/MappingWizard.js

```diff
@@ -190,7 +191,13 @@ class MappingWizard extends React.Component {
             onClick={onFinalStep ? () => hideMappingWizardAction(onFinalStep) : this.nextStep}
             disabled={disableNextStep}
           >
-            {onFinalStep ? __('Close') : activeStepIndex === 3 ? __('Create') : __('Next')}
+            {onFinalStep
+              ? __('Close')
+              : activeStepIndex === 3
+                ? editingMapping
+                  ? __('Save')
```

Comment:
Not seeing the `Save` text on the button while editing the mapping -- still seeing `Create`

Reply:
This is fixed by 2ee24e4